observer.js

```diff
@@ -1,71 +0,0 @@
-/*------------JS for fade in and slide in effects------*/
-
-const faders = document.querySelectorAll('.fade-in');
-const sliders = document.querySelectorAll('.slideIn');
-const sliderImg = document.querySelectorAll('.slideInImg');
-
-// Options for fade/slide effects
-
-const appearOptions = {
-    threshold: 0
-};
-
-// Set up for Intersection Observer
-
-const appearOnScroll = new IntersectionObserver(function(entries, appearOnScroll) {
```

Comment:
I've never heard of this IntersectionObserver. Cool way to set up animations on scroll